src/Test/commentCounter.test.js

```diff
@@ -0,0 +1,32 @@
+/**
+ * @jest-environment jsdom
+ */
+
+import commentCounterFunction from '../modules/commentCounter.js';
+
+describe('Comment Counter', () => {
```

Comment:
- [ ] I think your provided edge cases for the _counter_ function are very limited. I'll urge you to do some more search and come up with all the possible edge cases you can think about 💪😃